libs/platform/experience/src/routes.ts

```diff
@@ -42,6 +43,13 @@ export const defaultExperienceRoutes: RouteConfig[] = [
   {
     path: '/:page',
     type: RouteType.Page,
+    afterEnter: ({ page }) =>
```

Comment:
Just wondering, do we need `afterEnter` tied to router configuration, if we can just provide router event listener instead. Feels more decoupled, and easier to override/extend.